cdk-codepipeline-cross-account-deployment/README.md

```diff
@@ -0,0 +1,100 @@
+# CDK CodePipline Cross-Account Deployment
+
+This pattern deploys a CodePipline using CodeCommit and CodeBuild. From this pipeline it deploys a CloudFront distribution in front of an S3 Bucket to serve as a static website, across 2 accounts. This is an example of how you can use one pipline to deploy the same stack across multiple environments.
+
+Note that CloudFront and S3 are used in this pattern for demonstration purposes, you can use this pattern with any stack you wish to deploy.
+
+![Pattern architecture](img/serverless-pattern.png)
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the AWS Pricing page for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create AWS account(s)](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
```

Comment:
do they need to have special permissions to do the cross account? if so please specify

Reply:
No additional permissions are required outside of the general requirements required for bootstrapping an environment. I added more info on the next commit about this.